_posts/2018-07-12-open-source-design-nyc.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: event
+title:  "Open Source Design NYC: July 12"
+date:   2018-07-12
+categories: design meetup community
+eventDate: Thu, 12 July 2018
+location: New York
+time: 6:00pm - 8.00pm (EDT)
+status: past
```

Comment:
Should have been upcoming, but I guess since it's a relative short period, you put it automatically on 'past' :)